fastlane/report.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<testsuites>
+  <testsuite name="fastlane.lanes">
+    
+    
```

Comment:
nit: Remove extra lines

Reply:
Actually, this file shouldn't have been committed at all, that was just from my running the job locally on my machine. I'll delete it